From d92c57d5c45edaa4e11c29fe54a1c2fd2f2edb4a Mon Sep 17 00:00:00 2001
From: Haitao Huang <4699115+haitaohuang@users.noreply.github.com>
Date: Thu, 27 Feb 2020 12:40:25 -0600
Subject: [PATCH 2/5] Fix warning on 5.6 kernel

Signed-off-by: Haitao Huang <4699115+haitaohuang@users.noreply.github.com>
---
 sgx_main.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sgx_main.c b/sgx_main.c
index 36d9d1d..170dc8a 100644
--- a/sgx_main.c
+++ b/sgx_main.c
@@ -88,9 +88,9 @@ MODULE_VERSION(DRV_VERSION);
 #ifndef X86_FEATURE_SGX
 	#define X86_FEATURE_SGX (9 * 32 + 2)
 #endif
-
+#ifndef FEAT_CTL_SGX_ENABLED
 #define FEAT_CTL_SGX_ENABLED                      (1<<18)
-
+#endif
 /*
  * Global data.
  */
-- 
2.26.2

