From b34df8936a95fc619c68ea7bf7bae253b65edf8e Mon Sep 17 00:00:00 2001
From: Zhang Lili Z <lili.z.zhang@intel.com>
Date: Thu, 30 May 2019 18:47:03 +0800
Subject: [PATCH 5/5] Fix configid offset in secs.

Signed-off-by: Zhang Lili Z <lili.z.zhang@intel.com>
---
 sgx_arch.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sgx_arch.h b/sgx_arch.h
index 1c77f5f..c90c11d 100644
--- a/sgx_arch.h
+++ b/sgx_arch.h
@@ -92,8 +92,8 @@ struct sgx_secs {
 	uint32_t mrenclave[8];
 	uint8_t reserved2[SGX_SECS_RESERVED2_SIZE];
 	uint32_t mrsigner[8];
-	uint32_t configid[16];
 	uint8_t	reserved3[SGX_SECS_RESERVED3_SIZE];
+	uint32_t configid[16];
 	uint16_t isvvprodid;
 	uint16_t isvsvn;
 	uint16_t configsvn;
-- 
2.21.0

